modules/graceful-shutdown.adoc

```diff
@@ -77,3 +77,13 @@ Cluster administrators are responsible for ensuring a clean restart of their own
 ====
 
 . Shut off any cluster dependencies that are no longer needed, such as external storage or an LDAP server. Be sure to consult your vendor's documentation before doing so.
++
+[IMPORTANT]
+====
+After you delete your deployed cluster on a cloud-provider platform, you can remove the cloud resources associated with your cluster. These cloud resources might be in use by other clusters in the same cloud account. To remove the cloud resources, which are not required by other clusters, you must have the required permissions in your cloud-provider account.
+====
++
+[NOTE]
+====
+If you delete cloud resources of a suspended VM, then {product-title} might not restore properly.
```

Comment:
@pneedle-rh I feel we can rewrite the important block to be more user action oriented i.e specifying dos and don'ts rather than possibility with our products.

> **Important:** When you deploy your cluster on a cloud-provider platform, do not shut down or suspend the cloud resources associated with your cluster. If you delete cloud resources of a suspended VM, {product-title} might not restore successfully.

Let me know your thoughts on this.

Reply:
@darshan-nagaraj hi! I think that is a great idea. I have a couple of suggestions about your updated admonition.

1. You could change "When you deploy your cluster on a cloud-provider platform" to "If you deployed your cluster on a cloud-provider platform", because in context to the procedure, this has already occurred.

2. You could minimise the first sentence by removing the second reference to "your cluster", as follows:

> If you deployed your cluster on a cloud-provider platform, do not shut down or suspend the associated cloud resources.

Reply:
@pneedle-rh Fixed.